src/web/routes/application/enter-code/enter-code.js

```diff
@@ -28,13 +31,26 @@ const behaviourForGet = (req, res, next) => {
   next()
 }
 
+const behaviourForPost = (req, reqs, next) => {
+  if (validationResult(req).isEmpty()) {
+    req.session[CONFIRMATION_CODE_ENTERED_SESSION_PROPERTY] = true
```

Comment:
I think in later PR's we'll want to replace this (or add to it) with information about which email address/phone number was validated.

Reply:
We could log that information, not sure how much value it has in the session

Reply:
I think we may need it when deciding whether we need to send a code out again (they've gone back several steps, and clicked continue through the email and mobile number pages again, but did they make changes?). We may also need to store that information in the db with the claim (I'm not sure of the need for this yet). Either way, it's not required for this PR.

Reply:
We do have a story to cover the user changing their email/phone number. There we would invalidate the sending of the code if they changed the channel used for the code.